### ISSUE TITLE
Session key encoding and decoding

### ISSUE THREAD

**Issue by rfleschenberg** [awesto/django-shop#311]
I think we have a bug in the session key encoding / decoding (`CustomerManager.encode_session_key` and friends). I don't fully understand the issue yet, but there seems to be some indeterministic behaviour here, judging from the build failures on Travis. We should take a close look at these functions. The problem may be related to leading zeros in the username, maybe -- but I am not sure yet.

**Comment by rfleschenberg** [comment]
@jrief found the bug:

```
>>> int('123456789abcdef'[:32], 36)                                                                        
6496866212201143579095L
>>> int('0123456789abcdef'[:32], 36)                                                                       
6496866212201143579095L
>>> int('00000123456789abcdef'[:32], 36)                                                                   
6496866212201143579095L
>>> 
```

**Comment by rfleschenberg** [comment]
Fixed in PR #312.
